up.ps1

```diff
@@ -1,12 +1,15 @@
+[CmdletBinding(DefaultParameterSetName = "no-arguments")]
+Param (
+    [Parameter(HelpMessage = "Starts the rendering containers for SUGCON sites.")]
+    [switch]$IncludeSugconSites
+)
+
 # ENSURE ENV FILE EXISTS
 if(-not (Test-Path .env)) {
     Write-Host "Docker Env file not found. Have you run init.ps1 first? Refer to README.md for details" -ForegroundColor Red
     exit
 }
 
-# Comment the following line in case if you want to manually switch to windows container!
- & "C:\Program Files\Docker\Docker\DockerCli.exe" -SwitchWindowsEngine
-
```

Comment:
Can this be added back in?

Reply:
I removed it because it for some reason makes the next docker-compose call fail on my machine - see below screenshot.

Running Docker Desktop 4.3.2 (72729), WSL 2 based engine. It fails both on my desktop and laptop. It could be that the Daemon is still working on the switch when compose is called - and the switch is irrelevant since I am already running windows containers.

It also lack a Test-Path before making the call. So even if it did work, it would fail for the few users that use a non-default install dir. 

I would presume that there is some way to check which daemon is active (perhaps by reading config files). This could be a different approach to "solve" the same thing -I haven't looked into it. 

Imho I see it as very reasonable basic prerequisites that 1) docker is installed and running 2) docker is set to Windows containers.

![image](https://user-images.githubusercontent.com/2105432/150950598-bd29362a-5ef0-48d8-ab85-d5ba27394a83.png)

Reply:
Yeah that's fair, lets leave it out.